packages/ember-testing/lib/helpers.js

```diff
@@ -55,6 +55,7 @@ asyncHelper('visit', visit);
 
   @method click
   @param {String} selector jQuery selector for finding element on the DOM
+  @param {Object} context A DOM Element, Document, or jQuery to use as context
```

Comment:
I don't _personally_ care too much but seemingly using `{Object}` is basically just saying "I dunno" or "any" so not really helpful.